erlang/crypto_tests/sign.erl

```diff
@@ -0,0 +1,42 @@
+-module(sign).
+
+-export([cmd/0]).
+
+-include_lib("stdlib/include/assert.hrl").
+-include_lib("public_key/include/public_key.hrl"). 
+
+read_key(private, Name) ->
+    {ok, PemBin} = file:read_file(Name),
+    [RSAEntry] = public_key:pem_decode(PemBin),
+    Key = public_key:pem_entry_decode(RSAEntry),
+    Key;
+
+read_key(public, Name) ->
+    {ok, Bin} = file:read_file(Name),
+    [{Key, _}] = public_key:ssh_decode(Bin, public_key),
+    % [{Key, _}] <- spent like literally an hour figuring that one out, so stupid ಠ_ಠ
+    Key.
+
+
+cmd() ->
```

Comment:
move to unit test D:?

Reply:
oh wait I see this is a unity test D:

Reply:
not really, just me messing with some code to see if it works

Reply:
but yeah, i can make a unit test